src/caja-spatial-window.c

```diff
@@ -64,6 +64,7 @@
 
 #if !GTK_CHECK_VERSION(3,0,0)
 #define gtk_widget_get_preferred_size(x,y,z) gtk_widget_size_request(x,y)
+#define gtk_vbox_new(X,Y) gtk_box_new(GTK_ORIENTATION_VERTICAL,Y)
```

Comment:
Are sure?, this is what we use all over the code and not the new attempt from Sorkin :)
Still wondering that something which is used since 2 years in our code is bad now :)